ci/scripts/test-concourse-runner.sh

```diff
@@ -23,6 +23,17 @@ https://maven.apache.org/xsd/settings-1.0.0.xsd">
 EOF
 
 ## Test docker by running app tests
-mvn --global-settings settings.xml clean verify
+mvn --global-settings settings.xml clean verify --no-transfer-progress
+
+## Test docker scout
+echo "Checking docker scout installation..."
+DOCKER_SCOUT_VERSION=$(docker scout version | grep "darwin/amd64")
```

Comment:
This won't be the same version when it's installed by concourse, darwin is the macos platform. 

You can probably make this whole test just

```
if docker scout version >>/dev/null 2>&1; then
  echo "Docker scout is installed"
else
  ...
```

Reply:
Thanks! updated